src/app/components/turnos/dar-turnos/dar-turnos.component.ts

```diff
@@ -1256,7 +1257,9 @@ export class DarTurnosComponent implements OnInit {
             operacion = this.serviceListaEspera.post(listaEspera);
             operacion.subscribe();
         }
-
+        if (this.solicitudVacunacion) {
```

Comment:
Atención con esta función. `this.solicitudVacunacion` ya se evalúa en la linea 1218 en esta función. Creo que antes de seguir modificando esta función estaría bueno sentarse a revisarla un poco para acomodar código. Tiene varias cositas para mejorar. Cuantas veces dije función sin contar la última vez que dije función.